src/Config/RuntimeConfigLoader.cs

```diff
@@ -116,6 +74,9 @@ public static bool TryParseConfig(string json, [NotNullWhen(true)] out RuntimeCo
         return true;
     }
 
+    /// <summary>
+    /// Get Serializer options for the config file.
+    /// </summary>
     public static JsonSerializerOptions GetSerializationOptions()
```

Comment:
Since we don't use the JSON serializer in the base class, we should lift that up to the `FileSystemRuntimeConfigProvider`.

Reply:
We use the Json serializer options in the TryParseConfig method in the base class.